app/Models/Game.php

```diff
@@ -367,27 +346,29 @@ public function getImageIngameUrlAttribute(): string
         return media_asset($this->ImageIngame);
     }
 
-    public function getIsSubsetGameAttribute(): bool
+    public function getParentGameIdAttribute(): ?int
     {
-        if (str_contains($this->title, "[Subset")) {
-            return true;
-        }
-
         // Get all achievement sets associated with this game.
         $achievementSets = GameAchievementSet::where('game_id', $this->id)
             ->pluck('achievement_set_id');
 
         if ($achievementSets->isEmpty()) {
-            return false;
+            return null;
         }
 
         // Check if any of these achievement sets are used in other games with non-core types.
-        $hasNonCoreUsages = GameAchievementSet::whereIn('achievement_set_id', $achievementSets)
+        $nonCoreUsage = GameAchievementSet::whereIn('achievement_set_id', $achievementSets)
             ->where('game_id', '!=', $this->id)
             ->where('type', '!=', AchievementSetType::Core)
-            ->exists();
+            ->select('game_id')
+            ->first();
```

Comment:
Is it possible for a subset to be associated to multiple base games?

Reply:
Yes, such as for TLoZ: Oracle of Ages and TLoZ: Oracle of Seasons.

Reply:
Interesting. And it doesn't appear like that can find the parent on prod because the title doesn't match:
![image](https://github.com/user-attachments/assets/06a2b1e1-244e-4c78-8edb-5e90f05327e2)
![image](https://github.com/user-attachments/assets/558c5dc7-93b2-47fc-b3cb-5403754b38bb)

This code also can't find it because neither game has it linked as a subset, which leads back to questioning if we're at a point where `when game_achievement_sets is ready`.

What sort of `orderBy` should I add to make the parent consistent?

Reply:
> which leads back to questioning if we're at a point where when game_achievement_sets is ready.

I'm not sure how `game_achievement_sets` could handle this differently than it already is (assuming the subset is attached to both games properly).

I think in this case, we can't possibly have a breadcrumb that's correct because we're effectively fanning-in from two distinct games.

> What sort of orderBy should I add to make the parent consistent?

Good question - maybe `game_achievement_sets.created_at`? That's all that's coming to mind, though.